homeassistant/components/recorder/migration.py

```diff
@@ -40,6 +40,8 @@
 )
 from .util import session_scope
 
+LIVE_MIGRATION_MIN_SCHEMA_VERSION = 25
```

Comment:
We should probably merge this will a large number an bump it back down when we implement the migration of data